### PR TITLE
support circle ci integration and show circle ci status in README

### PR DESCRIPTION
**Pull request by makocchi-git**
- add circle.yml and support coverage by nosetests
- pep8 style check
- add circleci badge and license badge in README

### CODE REVIEW

**Comment by JumpeiArashi** [comment]
+1

**Comment by JumpeiArashi** [comment]
I'll check by today(I have a lot of time because I have a lot of MTG today :neckbeard:  )

File: blackbird/utils/configread.py [hunk 1]
```diff
@@ -661,7 +663,8 @@ def is_log(value):
 
            Recommended giving he full path including the file name.
 
-    :param str value: log file path. You can specify not only absolute path but also relational path.
+    :param str value: log file path. You can specify not only absolute path
+           but also relational path.
```
Comment (JumpeiArashi) [inline blackbird/utils/configread.py]: Sphinx style indent is four. You man write this param description below.

1. 
```
:param str value: log file path. You can specify not only absolute path
    but also relational path.
```

2. 
```
:param str value:
    log file path. You can specify not only absolute path
    but also relational path.
```

I'd like to style2.

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+dependencies:
+  override:
+    - python setup.py develop
+    - pip install coverage
+
+test:
+  override:
+    - pep8 blackbird/*.py
+    - pep8 blackbird/**/*.py
```
Comment (JumpeiArashi) [inline circle.yml]: L14 and L15 duplicate meaning?? Sorry if different meaning.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 blackbird
 =========
 
+[![Circle CI](https://circleci.com/gh/Vagrants/blackbird/tree/master.svg?style=shield)](https://circleci.com/gh/Vagrants/blackbird/tree/master)
+[![License](https://img.shields.io/badge/license-WTFPL-blue.svg)](http://www.wtfpl.net/about/)
+
```
Comment (JumpeiArashi) [inline README.md]: See https://github.com/Vagrants/blackbird/pull/44/files#diff-04c6e90faac2675aa89e2176d2eec7d8R130

Hmm... About License to write in two place.
I think that removing License at end of README, but I also think either is fine＼(^o^)／

**Comment by JumpeiArashi** [comment]
@makocchi-git 
I've just comments. Please see those.

File: circle.yml (hunk 2 above)
Comment (makocchi-git) [inline circle.yml]: `blackbird/**/*.py` means `blackbird/*/*.py`, `blackbird/*/*/*.py` ... so this is not including `blackbird/*.py`.
But I think that `pep8 blackbird/*.py blackbird/**/*.py` or `pep8 **/*.py` is more pretty.
which style do you like?

File: README.md (hunk 3 above)
Comment (makocchi-git) [inline README.md]: Oh.. certainly WTFPL badge is dupuricated !
which badge do you like?
I like blue one.
* WTFPL license logo
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)
* [License](http://www.wtfpl.net/about/)

File: circle.yml (hunk 2 above)
Comment (JumpeiArashi) [inline circle.yml]: I understood it.
For now, `pep8 **/*.py` is seems to be good.

File: README.md (hunk 3 above)
Comment (JumpeiArashi) [inline README.md]: Wow! amazing icons!
OK, remove license record at end of README and use below icon style(such as circle build icon).
coloring.... Hmm... I'd like to blue,too!
Thanks for good information about license record.

File: blackbird/utils/configread.py (hunk 1 above)
Comment (makocchi-git) [inline blackbird/utils/configread.py]: Got it. I'd like to style 2 too.
btw, I suggest style 3.

### 3.
```
:param str value:
    log file path
    You can specify not only absolute path but also relational path.
```
How do you think?

Comment (JumpeiArashi) [inline blackbird/utils/configread.py]: Oh, I said it about indent style.
style 3 seems to be good.

**Comment by JumpeiArashi** [comment]
LGTM:+1:

**Pull request merged**